app/src/main/java/com/androiddevs/news/extension/Extensions.kt

```diff
@@ -0,0 +1,79 @@
+package com.androiddevs.news.extension
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ProgressBar
+import android.widget.Toast
+import androidx.core.app.ActivityCompat
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LiveData
+import com.androiddevs.news.ui.LoginActivity
+import com.androiddevs.news.ui.NewsActivity
+import com.androiddevs.news.ui.RegistrationActivity
+
+fun Activity.displayToast(message: String) {
+    Toast.makeText(this.applicationContext, message, Toast.LENGTH_LONG).show()
+}
+
+fun Activity.displayToast(message: Int) {
+    Toast.makeText(this.applicationContext, message, Toast.LENGTH_LONG).show()
+}
+
+fun Fragment.displayToast(message: String) {
+    Toast.makeText(this.context, message, Toast.LENGTH_LONG).show()
+}
```

Comment:
Redundant "this"
If we already have an extension, better to write 
fun Fragment.displayToast(message: String) {
    context.displayToast(message)
}